ocdaction/ocdaction/dashboard_urls.py

```diff
@@ -22,6 +23,11 @@
         name="task-add"
     ),
     url(
+        r'^tasks/(?P<task_id>\d+)/$',
```

Comment:
can we change the url to have edit in it as we would also want to be able to view a task on this url. So if we can have the `^tasks/(?P<task_id>\d+)/edit$` for this view then the `^tasks/(?P<task_id>\d+)/$` is available for us for later to be able to view a task without editing